src/main/java/model/User.java

```diff
@@ -1,11 +1,23 @@
 package model;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class User {
 
     private Long id;
     private String email;
     private String password;
     private String role;
+    private List<Product> basket = new ArrayList<>();
```

Comment:
try to create separate entity Busket and work with it

Reply:
yes, i create Basket model